deepchem/data/tests/test_dftyaml.py

```diff
@@ -0,0 +1,9 @@
+from deepchem.data.data_loader import DFTYamlLoader
+import deepchem as dc
+
+def test_dftloader():
```

Comment:
The test file has 2 entries. There should be a check both entries are loaded correctly